Microsoft.Toolkit.Uwp.SampleApp/App.xaml.cs

```diff
@@ -83,6 +84,26 @@ public App()
             }
         }
 
+        /// <summary>
+        /// Event fired when a Background Task is activated (in Single Process Model)
+        /// </summary>
+        /// <param name="args">Arguments that describe the BackgroundTask activated</param>
+        protected override void OnBackgroundActivated(BackgroundActivatedEventArgs args)
+        {
+            base.OnBackgroundActivated(args);
+
+            var deferral = args.TaskInstance.GetDeferral();
+
+            switch (args.TaskInstance.Task.Name)
+            {
+                case "TestBackgroundTaskName":
```

Comment:
I know it is not your fault :) but could you please create a constant in the sample page for TestBackgroundTaskName and using it here instead of copying and pasting